src/data-services/__mocks__/weather-anomaly-data-service.js

```diff
@@ -1,11 +1,11 @@
-function getBaselineData() {
-    return new Promise((resolve, reject) => {
-       resolve({
-           data: [],
-       });
-    });
+export function getBaselineData() {
+  return Promise.resolve({
```

Comment:
While the use of `Promises` precedes this PR, we should continue to be aware the IE doesn't support them (even if we won't support IE in perpetuity). Either a note in user docs about supported browsers or JS polyfill is probably warranted. The former is probably easier :smile:

Reply:
I have a [polyfill PR for P2A](https://github.com/pacificclimate/plan2adapt-v2/pull/205) up now. If that works out, there's no reason not to apply it in all our projects.

As a side note, Promise is now absolutely bog standard, widespread, and very useful. IE's failure to support it is inexplicable but no reason to question using Promises.